MSVC compiler error "wallmounted_to_facedir": redefinition

"wallmounted_to_facedir" is declared as extern in minetest\src\util\directiontables.h(34) but it isn't actually extern. 

Original localized error message:
```
2>\Minetest\src\util\directiontables.cpp(113): error C2370: "wallmounted_to_facedir": Neudefinition; unterschiedliche Speicherklassen
2>\minetest\src\util\directiontables.h(34): note: Siehe Deklaration von "wallmounted_to_facedir"
```

--- a/src/util/directiontables.cpp
+++ b/src/util/directiontables.cpp
@@ -110,7 +110,7 @@ const v3s16 g_27dirs[27] =
 	v3s16(0,0,0),
 };
 
-constexpr const u8 wallmounted_to_facedir[6] = {
+const u8 wallmounted_to_facedir[6] = {
 	20,
 	0,
 	16 + 1,

